### DISCUSSION TITLE
.

### DISCUSSION THREAD

**Discussion by MrColoo** [coder/code-server#3566] [Q&A]
.

**Comment by vital987** [comment]
By seeing your pre-edited issue:
On local VS Code [Browser Preview](https://marketplace.visualstudio.com/items?itemName=auchenberg.vscode-browser-preview) extension can help but in code-server, there is no display server to display the browser, it's a direct web app, I will try to work on it, though the possibility is very low.

**Reply by vital987** [reply]
Yup, I've added a display server and brave browser to my [**project**](https://github.com/vital987/code-server), the extension works fine, excluding the brave debugger connection with the code-server so no live editing 🙁 so you need to refresh the page everytime you make a change.
